devbox/common/src/Util.scala

```diff
@@ -88,8 +90,13 @@ object Util {
       }
       .toMap
   }
+
   def sentryCapture(e: Throwable): Unit = {
     io.sentry.Sentry.getContext().addTag("whoami", System.getProperty("user.name"))
```

Comment:
I tried to reuse this with the `Event` but for some reason it didn't work. So I ended up adding it on the `EventBuilder` as well.